chia/consensus/blockchain.py

```diff
@@ -164,7 +169,9 @@ async def receive_block(
         block: FullBlock,
         pre_validation_result: Optional[PreValidationResult] = None,
         fork_point_with_peak: Optional[uint32] = None,
-    ) -> Tuple[ReceiveBlockResult, Optional[Err], Optional[uint32], List[CoinRecord]]:
+    ) -> Tuple[
+        ReceiveBlockResult, Optional[Err], Optional[uint32], Tuple[List[CoinRecord], Dict[bytes, List[CoinRecord]]]
```

Comment:
where is the call to this function where you do something with the additional return values?

Reply:
`receive_block_batch` and `respond_block` in ful_node.py call this function and then pass the results to `update_wallets`